src/styles/partials/components/_alt-list.scss

```diff
@@ -68,6 +68,18 @@ $icon-arrow-width: 32px;
   }
 }
 
+.dg_sidebar__section {
```

Comment:
@theilman7 i thought we were able to make these changes regardless of it being inside of the sidebar?

Reply:
@martypowell So the component works fine in its current state within the footer, maintaining the margins etc. we want. The problem comes when it is added to the sidebar, due to a reset string. The `<li>` and `<a>` tags are both selected in the reset string with `.dg_sidebar__section li` and `.dg_sidebar__section a`. This kind of "double dips" by adding `margin: 0 0 20px 0;` to both the `li `and `a `tags. The simplest methods I could think of to work around the reset are as followed. I could be missing something though, so let me know if there is a more efficient method to accomplish this.

1. Removing the `li `and `a `tag from the reset, which I think could have ripple effects, if there is something this is resetting that it should.

2. Adding a class to all the 'li' tags within the alt-list component. This allows the elements to be selected more accurately, but would add additional markup.

3. Selecting the elements within the sidebar component, as shown here. Thought this would be the most non-invasive method.